src/basic/generate-bpf-delegate-configs.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python3
+
+# SPDX-License-Identifier: LGPL-2.1-or-later
+
+# Convert the bpf_{cmd,map_type,prog_type,attach_type} enums into a string
+# array to be used as configuration options
+
+import re
+import sys
+
+def print_usage_and_exit():
+    print(f'Usage: {sys.argv[0]} code <header>')
```

Comment:
Not a matter, but `code` -> `header` is better? Not a strong opinion, feel free to ignore.